pymeasure/instruments/cisystems/sr200n.py

```diff
@@ -0,0 +1,181 @@
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+
+import struct
+from pymeasure.instruments import Instrument
+from pymeasure.instruments.common_base import CommonBase
+
+
+# - PARAMETER STRINGS - #
+TEMP = '0817bf'
+TEMP_STABLE = '0817ca'
+SET_POINT_MIN = '0817bc'
+SET_POINT_MAX = '0817bd'
+SET_POINT_GET = '0817c2'
+SET_POINT_SET = '0617be %f'
+CHOPPER_FREQ_GET = '081939'
+CHOPPER_FREQ_SET = '06193a %f'
+CHOPPER_POS = '06193b %i'
+
+
+class SR200N(Instrument):
+
+    # - temperature parameters ------------------------------ #
+    temp = CommonBase.measurement(
+        TEMP,
+        """Float representing the current temperature of the blackbody source.""",
+    )
+
+    temp_is_stable = CommonBase.measurement(
+        TEMP_STABLE,
+        """Boolean parameter to check if the temperature has stabilized.""",
+        cast=bool,
+    )
+    
+    set_point_min = CommonBase.measurement(
+        SET_POINT_MIN,
+        """Float representing the minimum temperature set point.""",
+    )
+    
+    set_point_max = CommonBase.measurement(
+        SET_POINT_MAX,
+        """Float representing the maximum temperature set point.""",
+    )
+    
+    set_point = CommonBase.control(
+        SET_POINT_GET, SET_POINT_SET,
+        """Float representing the blackbody temperature set point.""",
+    )
+
+    # - chopper parameters ---------------------------------- #
+    chopper_frequency = CommonBase.control(
+        CHOPPER_FREQ_GET, CHOPPER_FREQ_SET,
+        """Float representing the frequency of the internal chopper."""
+    )
+
+    chopper_position = CommonBase.setting(
+        CHOPPER_POS,
+        """Integer for manual control of the chopper (0 for closed, 1 for open)."""
+    )
+
+    # - internal parameters for constructing and reading messages - #
+    _header = b'\xaa\x01' 
+    _read_dict = {
+        TEMP: (14, float),
+        TEMP_STABLE: (11, bool),
+        SET_POINT_MIN: (14, float),
+        SET_POINT_MAX: (14, float),
+        SET_POINT_GET: (14, float),
+        CHOPPER_FREQ_GET: (14, float),
+    }
+    _read_count = 0
+    _read_type = None
+
+    def __init__(self, adapter, name='SR200N Blackbody', **kwargs):
+        super().__init__(
+            adapter,
+            name,
+            asrl={
+                'baud_rate': 115200,
+            },
+            **kwargs
+        )
```

Comment:
Please move the init at the top of the class definition.
And please add `includeSCPI=False`.